components/energylevel.stories.js

```diff
@@ -0,0 +1,8 @@
+import './energylevel.css'
+import level from './energylevel.html'
```

Comment:
Please keep the name. `level` is not a good name as it is much more than this.
```suggestion
import energylevel from './energylevel.html'
```